### PR TITLE
Js objects

### PR DESCRIPTION
**Pull request by MasumaJaffery**
In this Milestone;
1. Add Functionality to Add Objects Dynamically by JS.
2. Use of Constructor Function.
3. Add and Remove Objects from LocalStorage.
Thanks for your Time to Review my PR. 🤓

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @MasumaJaffery , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project with an unprofessional readme. Please fix the broken image link / remove it altogether. 
- [ ] Delete all instructions from your readme

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    // Add Event Listener to Add Button so if any User clicks the Book added as Object!
+    AddButton.addEventListener('click', () => {
+    // To Taking inputs from Users;
+      const Title = document.getElementById('title').value;
+      const Author = document.getElementById('author').value;
+      // The New Object Created with help of 'new' that is constructor func.
+      const book = new Book(Title, Author);
+      // LocalStorage Setup!
+      if (localStorage.length === 0) {
+        localStorage.setItem(i, JSON.stringify(book));
```
Comment (Ibizugbe) [inline index.js:30]: Nice job so far👍. Just as required in this milestone, kindly ensure your data is preserved in the local storage memory😉. if this is implemented, your page retains its rendered data (the books data) even when the page is reloaded😉. kindly ensure your data is preserved in local storage as required👍.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        // Object book Pushed(Added) in Books Array.
+        Books.push(book);
```
Comment (Ibizugbe) [inline index.js:32]: Currently, when you add or remove a book, it is updated in the UI👍. But the Books array is not updated with the added or removed book, kindly fix this👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Team👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are correctly setup and they are passing👍.
- [x] Github flow have been used👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] Kindly ensure you write descriptive and meaningful commit message as this practice is more professional👍.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @MasumaJaffery, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**